pootle/apps/import_export/templates/browser/index.html

```diff
@@ -31,15 +32,19 @@
   <form action="" method="post" enctype="multipart/form-data"
     id="js-upload-form" class="upload-form">
     {% csrf_token %}
-    {% for field in upload_form %}
-      {{ field }}
-    {% endfor %}
-    <input type="submit" />
+    <div class="hide">{{ upload_form.file }}</div>
+    <span {% if not has_admin_access %}class="hide"{% endif %}>
+    {% blocktrans with user=upload_form.user_id %}
+      as {{ user }}
```

Comment:
Personally I would have two strings:

- `<a>Upload translations</a>`
- `<a>Upload translations</a> as {{user}}`

The first one is as previously. The second one would contain some placeholder (perhaps a `<span>` with some `id` where you can inject the dropdown). Have in mind that despite our languages might keep the dropdown at the end of the phrase, other languages might require to have the dropdown in the middle of the phrase, thus the placeholder (a simple and short one) must be present within the translatable string to allow translators to reposition it if needed. Some translation comment might be necessary to help translators as well.